lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystem.cs

```diff
@@ -144,8 +148,10 @@ public IFileSystem GetAzureFileSystem()
                 injector.BindVolatileInstance(TestCloudBlobClient);
                 var fs = injector.GetInstance<AzureBlockBlobFileSystem>();
                 TestCloudBlobClient.BaseUri.ReturnsForAnyArgs(FakeUri);
+                TestCloudBlockBlob.Open().Returns(TestStream);
```

Comment:
I would put this logic in the TestOpen() function, but this is what I wanted. Thank you for adding it.